modules/social_features/social_group/modules/social_group_request/social_group_request.links.task.yml

```diff
@@ -0,0 +1,5 @@
+social_user.groups:
```

Comment:
@rochek03 Since we already use this ID, I think it's better to rename it to some other value, for example: `social_group.requests:`

Als, since it use existing ID  it affects and hides User Group tab:

<img width="1586" alt="image" src="https://github.com/user-attachments/assets/f6a7f83f-8628-4a3a-8840-4e992fad077c" />

Reply:
@ribel fixed